From f5b373e80c09ab29a9548432aa912f682e8023e6 Mon Sep 17 00:00:00 2001
From: "Christopher R. Palmer" <crpalmer@gmail.com>
Date: Wed, 27 Aug 2025 06:08:35 -0400
Subject: .config: update modules to get subsystems booting

Note: the final problem I encountered was that anaconda
blacklisted qcom_q6v6_pas.ko in /etc/modprobe.d.  This
may have been because it was blacklisted in the kernel
command line on the initial boot.
---
 .config | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/.config b/.config
index f3fdca5aae1c..765eee4ae9f0 100644
--- a/.config
+++ b/.config
@@ -2414,7 +2414,7 @@ CONFIG_REGMAP_KUNIT=m
 CONFIG_REGMAP_I2C=y
 CONFIG_REGMAP_SLIMBUS=m
 CONFIG_REGMAP_SPI=m
-CONFIG_REGMAP_SPMI=y
+CONFIG_REGMAP_SPMI=m
 CONFIG_REGMAP_MMIO=y
 CONFIG_REGMAP_IRQ=y
 CONFIG_REGMAP_RAM=m
@@ -5175,12 +5175,12 @@ CONFIG_PINCTRL_SM6115=m
 # CONFIG_PINCTRL_SM8250 is not set
 # CONFIG_PINCTRL_SM8350 is not set
 CONFIG_PINCTRL_SM8450=m
-# CONFIG_PINCTRL_SM8550 is not set
-# CONFIG_PINCTRL_SM8650 is not set
-# CONFIG_PINCTRL_SM8750 is not set
-CONFIG_PINCTRL_X1E80100=y
+CONFIG_PINCTRL_SM8550=m
+CONFIG_PINCTRL_SM8650=m
+CONFIG_PINCTRL_SM8750=m
+CONFIG_PINCTRL_X1E80100=m
 # CONFIG_PINCTRL_TLMM_TEST is not set
-CONFIG_PINCTRL_QCOM_SPMI_PMIC=y
+CONFIG_PINCTRL_QCOM_SPMI_PMIC=m
 # CONFIG_PINCTRL_QCOM_SSBI_PMIC is not set
 CONFIG_PINCTRL_LPASS_LPI=m
 CONFIG_PINCTRL_SC7280_LPASS_LPI=m
@@ -5190,8 +5190,8 @@ CONFIG_PINCTRL_SM8250_LPASS_LPI=m
 CONFIG_PINCTRL_SM8350_LPASS_LPI=m
 CONFIG_PINCTRL_SM8450_LPASS_LPI=m
 CONFIG_PINCTRL_SC8280XP_LPASS_LPI=m
-# CONFIG_PINCTRL_SM8550_LPASS_LPI is not set
-# CONFIG_PINCTRL_SM8650_LPASS_LPI is not set
+CONFIG_PINCTRL_SM8550_LPASS_LPI=m
+CONFIG_PINCTRL_SM8650_LPASS_LPI=m
 
 #
 # Renesas pinctrl drivers
@@ -9968,7 +9968,7 @@ CONFIG_ARM64_PLATFORM_DEVICES=y
 CONFIG_EC_ACER_ASPIRE1=m
 # CONFIG_EC_HUAWEI_GAOKUN is not set
 CONFIG_EC_LENOVO_YOGA_C630=m
-CONFIG_EC_LENOVO_YOGA_SLIM7X=y
+CONFIG_EC_LENOVO_YOGA_SLIM7X=m
 CONFIG_HAVE_CLK=y
 CONFIG_HAVE_CLK_PREPARE=y
 CONFIG_COMMON_CLK=y
@@ -11805,9 +11805,9 @@ CONFIG_INTERCONNECT_QCOM_SM8150=m
 CONFIG_INTERCONNECT_QCOM_SM8250=m
 # CONFIG_INTERCONNECT_QCOM_SM8350 is not set
 CONFIG_INTERCONNECT_QCOM_SM8450=m
-# CONFIG_INTERCONNECT_QCOM_SM8550 is not set
-# CONFIG_INTERCONNECT_QCOM_SM8650 is not set
-# CONFIG_INTERCONNECT_QCOM_SM8750 is not set
+CONFIG_INTERCONNECT_QCOM_SM8550=m
+CONFIG_INTERCONNECT_QCOM_SM8650=m
+CONFIG_INTERCONNECT_QCOM_SM8750=m
 CONFIG_INTERCONNECT_QCOM_X1E80100=y
 CONFIG_INTERCONNECT_QCOM_SMD_RPM=m
 CONFIG_INTERCONNECT_CLK=y
-- 
2.51.0

